examples/table-queries-select-all/table_queries_select_all.description

```diff
@@ -0,0 +1,6 @@
+// Ballerina supports querying tables using a syntax similar to SQL. This query syntax can only be used to query a table.
+// Modification of an existing table is not supported.
+// When an existing table is queried using Ballerina SQL, a new in-memory table is created with the queried data and
+// that table is returned.
+// Executing these SQL-like queries on tables returned through Ballerina database clients is not
+// supported yet.
```

Comment:
The example table_queries_join also has the same description. Shouldn't we include some part to address the specific usage in this example in the descrption?